storage/libradb/src/backup/restore_handler.rs

```diff
@@ -122,4 +140,23 @@ impl RestoreHandler {
 
         self.db.write_schemas(cs.batch)
     }
+
+    pub fn get_tree_state(&self, num_transactions: LeafCount) -> Result<TreeState> {
+        let frozen_subtrees = self
+            .ledger_store
+            .get_frozen_subtree_hashes(num_transactions)?;
+        let state_root_hash = if num_transactions == 0 {
+            self.state_store
+                .get_root_hash_option(PRE_GENESIS_VERSION)?
+                .unwrap_or(*SPARSE_MERKLE_PLACEHOLDER_HASH)
```

Comment:
why not expect?

Reply:
? when there's no PRE_GENESIS_VERSION root, we need to fallback to the default hash.

Reply:
I think I am confused cuz I thought the `PRE_GENESIS_VERSION` will be always pre-created.